NorthstarDLL/squirrel.cpp

```diff
@@ -238,12 +238,16 @@ template <ScriptContext context> SQInteger SQPrintHook(HSquirrelVM* sqvm, const
 	return 0;
 }
 
-template <ScriptContext context> CSquirrelVM* (*__fastcall CreateNewVM)(void* a1, ScriptContext realContext);
-template <ScriptContext context> CSquirrelVM* __fastcall CreateNewVMHook(void* a1, ScriptContext realContext)
+template <ScriptContext context> CSquirrelVM* (*__fastcall CreateNewVM)(void* a1, int realContext);
```

Comment:
why are these ints now?

Reply:
because they were never ScriptContext's, it just so happened that the values overlapped for our usecase.